packages/manager/modules/octavia-load-balancer/src/delete/delete.controller.js

```diff
@@ -0,0 +1,44 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(atInternet, $http, Alerter, $translate) {
+    this.isLoading = false;
+    this.atInternet = atInternet;
+    this.$http = $http;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+  }
+
+  delete() {
+    this.isLoading = true;
+    return this.$http
+      .delete(
+        `/cloud/project/${this.serviceName}/region/${this.loadBalancerRegion}/loadbalancing/loadbalancer/${this.loadBalancerId}`,
+      )
+      .then(() => {
```

Comment:
Add the success Alert here. See https://github.com/ovh/manager/pull/10044/files#r1343787678